content/en/cosign/overview.md

```diff
@@ -15,98 +15,84 @@ features:
-
-![Cosign demo gif](/cosign.gif)
-
-Cosign supports:
+## Getting Started (Quick Start)
 
-<list :items="features" type="info"></list>
+Cosign supports blob and container signing, verification, and storage in an OCI registry.
+Cosign aims to make signatures invisible infrastructure.
 
 Cosign is part of the Sigstore project. Join us on our [Slack channel](https://sigstore.slack.com/) (need an [invite](https://links.sigstore.dev/slack-invite)?)
 
-## Getting Started (Quick Start)
-
-To get up and running we'll demonstrate how to: 
+### Installation
 
-1. Generate a keypair
-1. Sign a container image and store that signature in the registry
-1. Find signatures for a container image, and verify them against a public key
+To get sigstore up and running, you simply need to install Cosign. instructions to install Cosign can be found at https://docs.sigstore.dev/cosign/installation/. 
 
-### Prerequisites
+This will allow you to sign both blobs and containers.   If you need a sample container to test signing with, you can create one by following the instructions at: https://cloud.google.com/artifact-registry/docs/docker/store-docker-container-images
 
-You'll need to [install Cosign](/cosign/installation/) first, and you will need access to a container registry.
+### Signing a blob
 
-[ttl.sh](https://ttl.sh) offers free, short-lived (hours), anonymous container image
-hosting if you just want to try out these commands.
+The basic signing format for a blob is as follows:
 
-### 1. Generate a keypair
 
-```shell
-$ cosign generate-key-pair
-Enter password for private key:
-Enter again:
-Private key written to cosign.key
-Public key written to cosign.pub
 ```
-
-### 2. Sign a container and store the signature in the registry
-
-```shell
-$ cosign sign --key cosign.key user/demo
-Enter password for private key:
-Pushing signature to: index.docker.io/user/demo:sha256-87ef60f558bad79beea6425a3b28989f01dd417164150ab3baab98dcbf04def8.sig
+$ cosign sign-blob <file> --bundle txtbundle.bundle
 ```
 
-The `cosign` command above prompts the user to enter the password for the private key.
-The user can manually enter the password, or set an environment variable with `COSIGN_PASSWORD` to use a password automatically.
-
-### 3. Verify a container against a public key
+The bundle contains signing metadata, including the signature and certificate.  
 
-This command returns `0` if *at least one* `cosign` formatted signature for the image is found
-matching the public key. Review the other sections of this site for information and caveats on other signature formats.
+The Cosign command requests a certificate from our Certificate Authority Fulcio. Fulcio checks your identity by using an authentication protocol (OpenID Connect) to look at your email address. If your identity is correct, Fulcio grants a short-lived, time-stamped certificate. The certificate is bound to the public key to attest to your identity.  This activity is logged using Rekor.
```

Comment:
```suggestion
The Cosign command requests a certificate from our Certificate Authority, which is called Fulcio. Fulcio checks your identity by using an authentication protocol (OpenID Connect) to look at your email address. If your identity is correct, Fulcio grants a short-lived, time-stamped certificate. The certificate is bound to the public key to attest to your identity.  This activity is logged using our Transparency Log, which is called Rekor.
```

Reply:
* lower case certificate authority and transparency log
* change `our` to `the`